qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -138,20 +151,24 @@ def _prepare_operator(
             Tuple[ParameterExpression, ParameterExpression],
             List[Tuple[ParameterExpression, ParameterExpression]],
         ],
+        aux_meas_op: OperatorBase = Z,
     ) -> OperatorBase:
         """Traverse ``operator`` to get back the adapted operator representing the gradient.
 
         Args:
             operator: The operator we are taking the gradient of: ⟨ψ(ω)|O(θ)|ψ(ω)〉
             params: The parameters we are taking the gradient wrt: ω
-                    If a ParameterExpression, ParameterVector or List[ParameterExpression] is given,
-                    then the 1st order derivative of the operator is calculated.
-                    If a Tuple[ParameterExpression, ParameterExpression] or
-                    List[Tuple[ParameterExpression, ParameterExpression]]
-                    is given, then the 2nd order derivative of the operator is calculated.
+                If a ``ParameterExpression```, ``ParameterVector`` or ``List[ParameterExpression]``
+                is given, then the 1st order derivative of the operator is calculated.
+                If a ``Tuple[ParameterExpression, ParameterExpression]`` or
+                ``List[Tuple[ParameterExpression, ParameterExpression]]``
+                is given, then the 2nd order derivative of the operator is calculated.
+            aux_meas_op: The operator that the auxiliary qubit is measured with respect to.
+                 For aux_meas_op = Z we compute Re[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉].
+                 For aux_meas_op = -Y we compute Im[(dω⟨ψ(ω)|)O(θ)|ψ(ω)〉].
```

Comment:
This misses the case `Z - iY`, also could you use the code-quotes for the variable names like you did above? 🙂